scripts/build/docker/Dockerfile.template

```diff
@@ -16,6 +16,8 @@ RUN apt-get update \
     curl \
     fuse \
     git \
+    graphicsmagick \
```

Comment:
Even `gcc-multilib` can't stop a not-cross-compilation-aware `configure` script setting things up wrong ;-)

WRT testing, I meant that we test that a 32-bit image runs on a pure 32-bit host. I think if we compiled on a 64-bit host, and ended up with a '32-bit' image which had a mix of 32-bit and 64-bit parts, then trying to test it on a 64-bit host would still "work"?